queries.sql

```diff
@@ -1,3 +1,66 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = TRUE;
+
+SELECT * FROM animals WHERE name <> 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = 'null';
```

Comment:
[OPTIONAL]
- [ ] Nice work updating the table :clap: However it would be nice to add the command `COMMIT` to save the changes that were made.

_**This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you make the change**_

Reply:
thanks